app/brave_main_delegate.cc

```diff
@@ -172,6 +172,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
   // Disabled features.
   const std::unordered_set<const char*> disabled_features = {
       autofill::features::kAutofillServerCommunication.name,
+      blink::features::kTextFragmentAnchor.name,
```

Comment:
This works as well. Tested using `https://example.com/#:~:text=domain` 

This needs to be added to: https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove) - once the PR is merged.

Reply:
@jumde added to Wiki.

Reply:
Thanks @mkarolin